backend/baseclass.pm

```diff
@@ -1241,6 +1242,8 @@ sub new_ssh_connection {
     }
 
     my $ssh = Net::SSH2->new;
+    # timeout requires libssh2 >= 1.2.9 so not all versions might have it
+    $ssh->timeout($bmwqemu::vars{SSH_COMMAND_TIMEOUT_S} * 1000) if $ssh->can('timeout');
```

Comment:
I suppose the `timeout` method will actually always be available as it is provided by the Perl module and not the underlying libssh2 library. I would just set the option via the constructor:

```suggestion
    # timeout requires libssh2 >= 1.2.9
    my $ssh = Net::SSH2->new(timeout => ($bmwqemu::vars{SSH_COMMAND_TIMEOUT_S} // 10 * ONE_MINUTE) * 1000);
```